### ISSUE TITLE
From Antarctica everywhere is North!

### ISSUE THREAD

**Issue by Gcc9876** [teuteuf/worldle#310]
Surprised to see China referenced as being NE of Antarctica (my first, incorrect guess). Does Worldle not use great circle tracks?
![3CB55B1D-EFE6-4EFD-9B90-430B2CD6C8F2](https://user-images.githubusercontent.com/111962663/186362365-60155f4b-1d82-49c7-8a6b-ad5265be0d86.png)

**Comment by DavidLloydOwen** [comment]
My guess would be teuteuf measures distances between countries and directions between them using the centres of gravity of the country silhouettes. The CoG of Antarctica is unlikely to be exactly at the Pole, hence not all other countries are directly North.

**Comment by Gcc9876** [comment]
Agreed, but for the continents and distances involved I would have thought it would still be way closer to north than north-east?   Y suspicion is that it’s using a Mercator style projection, which may cause issues for our North American cousins whose Mercator maps tend to have the americas centred within them! For them, China would ne north-west of Antarctica :-)
